website/content/en/docs/migration/version-upgrade-guide.md

```diff
@@ -1060,15 +1060,15 @@ RUN ansible-galaxy collection install -r ${HOME}/requirements.yml \
  && chmod -R ug+rwx ${HOME}/.ansible
 ```
 
-## Unreleased (Master Branch)
+## v0.17.x
 
-**modules**
+### modules
```

Comment:
I think we need to revert these bold -> header changes.


IIRC we intentionally changed these from headers to bold, because Hugo renders links for headers, and we didn't want links for each section of the migration guide.

Reply:
Hun .. OK I will revert. However, IHMO the links were better for we are able to address specific points to users.